scripts/maixbit-flash.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# @file
+# @brief Load firmware to MAiX BiT board
+#		Requires kflash.py(https://github.com/kendryte/kflash.py)
+# @author	sksat
+# @date		05.08.2020
+#
+
+BUILD_DIR=./build/base/bin
+EM_BIN=$BUILD_DIR/embox.bin
+
+KFLASH=/usr/bin/kflash
```

Comment:
Why do you declare KFLASH, you don't use it as I see
and in my system kflash locate in other place
```
which kflash 
/usr/local/bin/kflash
```

Reply:
I thought it would be nice to be able to specify kflash location as env, but no problem if kflash installed correctly and the location is in PATH. I will remove it.

Reply:
Yes, you right it is nice to specify the location. it will be able to be added in the future